vsphere/datadog_checks/vsphere/data/conf.yaml.example

```
@@ -21,6 +21,21 @@ instances:
     #
     password: <PASSWORD>
 
+    ## @param use_legacy_check_version - boolean - required
+    ## For backward compatibility reasons, it is possible to use a deprecated version of the vsphere integration by
+    ## setting this field to "true".
+    #
+    use_legacy_check_version: false
+
+    ## @param empty_default_hostname - boolean - optional - default: false
+    ## The vSphere integration is a cluster-level check where metrics are usually unrelated to the host
+    ## on which the agent runs. Setting this parameter to true, prevents the agent from attaching the hostname
```

(Unrelated case nit)

```suggestion
    ## on which the agent runs. Setting this parameter to true, prevents the Agent from attaching the hostname
```